### ISSUE TITLE
Lorenz issue with Param1 knob

### ISSUE THREAD

**Issue by The-Great-Assyr** [baconpaul/mschack-VCV-Rack-Plugins#25]
Hello there and friendly greetings again!

First: sorry if I have not answered to my previous issue, I'll answer now if it is still useful.
It does not always draw out of bounds, only at certain parameters, like:
P1 = DEFAULT           T1 = DEFAULT
P2 = DEFAULT           T2 = 0
P3 = 0.132                T3 = 0.446
All the rest as DEFAULT

Another problem is that when all the parameter are DEFAULT, if I turn parameter P1 = 0 it seems that the whole module stop to work. I can twist any other knobs but I got no changes at all, both visually and audio. I have to CTRL+R the module to make it work again. I don't know if it is intended or if it is a bug.

If I turn
P1 = 1
P2 = 0.946
P3 = 0.095
All the T as DEFAULT, all the Filters as DEFAULT and all the Levels to the max (X = Y = Z = 10) it happens what you can see in the picture and consider that the cyan curve is static, not moving and there is no output sound I can hear.

All in all it seems (but I might be wrong, not knowing how the algorithm work) that the P1 knob/parameter work a little bit weird.

![capture_001_28082023_223250](https://github.com/baconpaul/mschack-VCV-Rack-Plugins/assets/35512040/727e8c0d-1c5f-4511-bf9d-302e06d243b5)

**Comment by Petervos2018** [comment]
Hi Andrea, 
Paul will come and tell you that he solved the out of screen rendering in the latest nightly, and I will tell you that the V1 version of the module reacted the same way , it's a very volatile module, it breaks up very easily, and Paul has not really changed any of the code.

**Comment by baconpaul** [comment]
yeah so the lorenz attractor (math) has plenty of parts of the space which are just collapsed to a single point not an orbit and that's what is happening here I presume. The parameters seem a bit mis-calibrated to me also but I'm hesitant to solve them because of other folks patches.

I suppose I could check a streaming version or some such thing and adjust them but that's more an "after 2.0.0" type thing.
